package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless",
-  "version": "0.5.5",
+  "version": "1.0.0",
```

Comment:
Opened up a separate issue for this task: https://github.com/serverless/serverless/issues/1363